README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/intel/sriov-cni.svg?branch=master)](https://travis-ci.org/intel/sriov-cni) [![Go Report Card](https://goreportcard.com/badge/github.com/intel/sriov-cni)](https://goreportcard.com/report/github.com/intel/sriov-cni)
```

Comment:
same comment on changing to travis-ci.org/k8snetworkplumbingwg

Reply:
I will send a new revision fixing it. Thanks